### ISSUE TITLE
Running multiple instances with the same whatsapp user

### ISSUE THREAD

**Issue by xowos** [pedroslopez/whatsapp-web.js#918]
### Description of the bug
I have a whatsapp application with a conversation logic running constantly and correctly, then I start another whatsapp application with the same user in whatsapp (same wa web connection) but only to send messages, then my first application stops working and restarts.

### Steps to reproduce
1. Run first application (chatbot - whatsapp instance), with a logic in watson assistant, to be able to answer messages correctly, via pm2.
2. Run second application (same whatsapp instance), only to send messages, no conversation logic, does not use pm2.
My first application is disconnected and restarted.
...

### Expected behavior 
My first application continues to run even though the second application is running for a few moments (2 to 3 minutes approximately).

### Environment (please complete the following information):
**WhatsApp**
 - Account type: Standard
 - Device operating system: Android
 - WhatsApp Web Version [run `await client.getWWebVersion()`]: 2.2138.13

**Library**
 1 application
- Browser: Chromium
 - NodeJS version (`node -v`): 14.17.3
 - npm or yarn version (`npm -v`): 6.14.13
 - whatsapp-web.js version: 1.14.0

2 application
- Browser: Chromium
 - NodeJS version (`node -v`): 14.17.3
 - npm or yarn version (`npm -v`): 6.14.13
 - whatsapp-web.js version: 1.14.1

**Other**
 - Operating system: Windows
 - Operating system version: w10

### Additional context.
This happens because whatsapp only allows one user to be logged in.

**Comment by KeruMx** [comment]
no, only in the whatsbeta, but i think you don need two clients running, one client, same session, just change your logic.

**Comment by xowos** [comment]
so with the multi-device version that is in beta version can it be possible?

I understand, I was saying this because the first application is based on responding and the second application is based on just sending messages with results and I could not make the first application send messages dynamically, since these messages depend on another application and that is why I decided to put it out of the first application.

**Comment by KeruMx** [comment]
yeah, with that version you can run 4 instances.
i think is better only once instance jeje

**Comment by xowos** [comment]
I understand, but for my needs it would be better to have several instances, because if the messages I have to send are too many, several instances can work at the same time and send the results

**Comment by PurpShell** [comment]
> yeah, with that version you can run 4 instances. i think is better only once instance jeje

yes

**Comment by PurpShell** [comment]
> I understand, but for my needs it would be better to have several instances, because if the messages I have to send are too many, several instances can work at the same time and send the results

i think that it wouldn't be of much help, except for using multiple threads on a cpu, since Node is single-threaded.

It won't be too much of an advantage
